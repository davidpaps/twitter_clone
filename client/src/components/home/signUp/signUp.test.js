import React from "react";
import { shallow } from "enzyme";
import SignUp from "./signUp";

describe("Sign Up", () => {
  let wrapper;

  beforeEach(() => (wrapper = shallow(<SignUp />)));

  it("should render a <div />", () => {
    expect(wrapper.find("div").length).toEqual(1);
  });

  it("should render a <img />", () => {
    expect(wrapper.find("img").length).toEqual(1);
  });

  it("should render a <form />", () => {
    expect(wrapper.find("form").length).toEqual(1);
  });

  it("should render a <Input/>", () => {
    expect(wrapper.find("input").length).toEqual(3);
  });

  it("should render a <button/>", () => {
    expect(wrapper.find("button").length).toEqual(2);
  });

  it("should render a <Link />", () => {
    expect(wrapper.find("Link").length).toEqual(1);
  });

  it("should render correctly", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
