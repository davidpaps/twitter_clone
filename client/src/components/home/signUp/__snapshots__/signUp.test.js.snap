// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sign Up should render correctly 1`] = `ShallowWrapper {}`;
